var/spack/repos/builtin/packages/py-snakemake-storage-plugin-gcs/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PySnakemakeStoragePluginGcs(PythonPackage):
+    """A Snakemake storage plugin for Google Cloud Storage."""
+
+    homepage = "https://github.com/snakemake/snakemake-storage-plugin-gcs"
+    pypi = "snakemake_storage_plugin_gcs/snakemake_storage_plugin_gcs-0.1.3.tar.gz"
+
+    license("MIT")
+
+    version("0.1.3", sha256="f0315596120160656b8c8afec66e3b31b4a2889b9d0cead2102f9d924ec0b326")
+
+    depends_on("py-google-cloud-storage@2.12:2", type=("build", "run"))
+    depends_on("py-google-crc32c@1.1.2:1", type=("build", "run"))
+
+    # This is not in the package definition, but I am pretty sure that it is needed
```

Comment:
Yep, it's imported in `__init__.py`. Might be worth reporting this bug to the developers

Reply:
Added an issue https://github.com/snakemake/snakemake-storage-plugin-gcs/issues/19

Reply:
Maybe replace your comment with a link to this issue.

Reply:
Added a link to issue.